src/eterna/mode/FeedbackViewMode.ts

```diff
@@ -102,7 +102,7 @@ export default class FeedbackViewMode extends GameMode {
 
         this._toolbar.screenshotButton.clicked.connect(() => this.postScreenshot(this.createScreenshot()));
         this._toolbar.viewSolutionsButton.clicked.connect(() => this.loadDesignBrowser());
-        this._toolbar.viewOptionsButton.clicked.connect(() => this.showViewOptionsDialog());
+        this._toolbar.settingsButton.clicked.connect(() => this.showViewOptionsDialog());
```

Comment:
I agree with this naming change, but could we carry it through farther? IE, showViewOptionsDialog -> showSettingsDialog (in all modes) and rename the class from EternaViewOptionsDialog -> EternaSettingsDialog.